src/services/mock/checkpoint.js

```diff
@@ -3,7 +3,7 @@ const checkpoint = {
     "workflows": [
         {
             "id": "user|one",
-            "name": "one",
+            "name": "one-two-three-four-five-fix-seven-eight-nine-ten-eleven-twelve-thirteen",
```

Comment:
Will undo this change later before asking for the final review, it just makes it easier for dev/testing for now :+1: